akvo/rsr/models/organisation.py

```diff
@@ -278,13 +277,7 @@ def knowledge(self):
         def all_projects(self):
             "returns a queryset with all projects that has self as any kind of partner"
             from .project import Project
-            return Project.objects.filter(partnerships__organisation__in=self).distinct()
-
-        def public_projects(self):
-            """
-            Returns a queryset with all public projects that has self as any kind of partner.
-            """
-            return self.all_projects().public()
+            return Project.objects.of_partner(self).distinct()
```

Comment:
Question on the underlying code: Since Project.of_partner() actually is Project.QuerySet.of_partner(), it should work with a queryset of organisations as param, right? But if so, isn't it identical in function to Project.of_partners()?